src/lib/options.js

```diff
@@ -0,0 +1,102 @@
+/*
```

Comment:
What's the reason for moving this logic to a separate file? Could we move any other logic here as well?

Reply:
I tried to document the reason in the commit comment for the commit where the move happened: db92e3d191255d22c129b2593457b49ed1a898a5. Let me know if the explanation makes sense.

We could definitely move other logic there (ideally the file in `js/` is a small "controller" while the file in `lib/` contains reusable functions), it's just a matter of improving the code while limiting changes to help avoid introducing new bugs (the less to review, the easier to understand) and allocating our time well.

Reply:
Oh yeah, that does make sense. I agree, we should limit other changes for now.